roles/maintenance_12_ubuntu/tasks/main.yml

```
@@ -0,0 +1,98 @@
+---
+
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
```

```suggestion
    taskid: ignore-me
```
see https://github.com/adfinis/ansible-collection-maintenance/pull/11